src/App.tsx

```diff
@@ -1,61 +1,114 @@
+import { useState } from 'react';
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+export const App: React.FC = () => {
+  const todosWithUsers = todosFromServer
+    .map(todo => {
+      const theUser = usersFromServer.find(user => user.id === todo.userId);
+
+      return theUser ? { ...todo, user: theUser } : null;
```

Comment:
It looks strange. If a todo doesn't have a user, should we not show the todo at all?
I think no